unlock-app/src/__tests__/utils/GlobalErrorProvider/missingProvider.test.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { Provider } from 'react-redux'
+import * as rtl from 'react-testing-library'
+import createUnlockStore from '../../../createUnlockStore'
+import { FATAL_MISSING_PROVIDER } from '../../../errors'
+
+jest.mock('../../../config', () =>
+  jest.fn().mockImplementation(() => {
+    return {
+      providers: {},
+      isRequiredNetwork: () => true,
+      requiredNetwork: 'dev',
+      services: { storage: { host: 'http://atest.url' } },
+    }
+  })
+)
+
+const {
+  default: GlobalErrorProvider,
+  GlobalErrorContext,
+} = require('../../../utils/GlobalErrorProvider')
+
+describe('GlobalErrorProvider', () => {
+  function makeTestStore(newValues = {}) {
+    return createUnlockStore(
+      {
+        account: {
+          address: '0xdeadbeef',
+          balance: '1000',
+        },
+        network: {
+          name: 1337,
+        },
+        router: {
+          route: '/somewhere',
+        },
+        ...newValues,
+      },
+      undefined,
+      true
+    )
+  }
+
+  // eslint-disable-next-line
+  function PeekAtContextConsumer() {
```

Comment:
nit: since this is not exported please use `peekAtContextConsumer`

Reply:
👍

Reply:
can't - React components must start with a capital letter, this is used as a React component in the test